docs/pages/database-access/auto-user-provisioning/database-access-controls.mdx

```diff
@@ -0,0 +1,210 @@
+---
+title: Database Access Controls
+description: Understand the concepts behind Database Access Controls managed by Teleport.
+---
+
+**Database Access Controls** is a Teleport feature that lets you configure
+role-based access controls for database objects. A **database
+object** can be a table, view, or stored procedure. With Database Access Controls, you can ensure that users only have permissions to manage the data
+they need.
+
+You can define **import rules** that instruct the Teleport Database Service to
+apply labels to database objects imported from databases that match labels configured within the import rule. When a user connects to a database, the Database Service selectively grants permissions by checking database object labels against the user's Teleport roles.
+
+The Database Service grants object-level permissions for the duration of a
+connection and revokes them automatically when the connection ends.
+
+## Database object import rules
+
+A database object import rule in Teleport is a resource that defines the labels to be applied to database objects imported into Teleport. If a specific object does not match any of the rules, it will not be imported.
+
+By default, if no import rules are present, Teleport will automatically create the `import_all_objects` rule on startup:
```

Comment:
> None of the clusters (upgraded or new) will have any import rules defined initially. Doesn't this effectively mean that the default rule will always be present? Might be helpful to clarify.

It will always be created if there are none. If you create an empty rule, it won't happen. If you modify the default rule, your modifications will prevail. If you delete the last rule you have, a new one will be created on startup.

> Also, is it created only when database access is enabled, or always?

Always.